build.sbt

```diff
@@ -13,7 +13,7 @@ scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-Xlint")
 // resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 
 libraryDependencies ++= Seq(
-  "com.algorithmia" % "algorithmia-client" % "1.0.+",
+  "com.algorithmia" % "algorithmia-client" % "1.0.15",
```

Comment:
Let's update this to be a version range where 1.0.15 is min and 2.0 is the exclusive max: `[1.0.15,2.0[` according to the [Ivy docs](https://ant.apache.org/ivy/history/2.3.0/ivyfile/dependency.html#revision) I believe. (Or similar version range format if that's not the proper syntax to use here.)

Assuming we follow semvar with the libraries we own :angel:, then all versions beneath 2.0 should be backwards-compatible with 1.0.15, 1.0.16, etc. That way we won't have to come back here to bump the `algorithmia-client` for each new patch or minor version release. We'll only need to bump for a major revision.